client/pkg/transport/listener.go

```diff
@@ -564,6 +652,10 @@ func (info TLSInfo) ClientConfig() (*tls.Config, error) {
 	return cfg, nil
 }
 
+func (info *TLSInfo) Close() {
```

Comment:
Aside of integration tests I didn't find any other places where this is called. I would expect EtcdServer to call this. Is this correct or maybe I missed it?

Reply:
fixed